example/project-native-restyle/styles/typography.json

```
@@ -0,0 +1,261 @@
+{
+  "400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "16px",
+    "fontSize": "12px",
+    "letterSpacing": "0.4px"
+  },
+  "500": {
+    "fontFamily": "Inter",
+    "fontWeight": 500,
+    "lineHeight": "24px",
+    "fontSize": "12.003px",
+    "letterSpacing": "0.1px"
+  },
+  "600": {
+    "fontFamily": "Inter",
+    "fontWeight": 600,
+    "lineHeight": "16px",
+    "fontSize": "12px",
+    "letterSpacing": "1.5px",
+    "textTransform": "uppercase"
+  },
+  "700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "16px",
+    "fontSize": "12px",
+    "letterSpacing": "1.25px",
+    "textTransform": "uppercase"
+  },
+  "h1-400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "112px",
+    "fontSize": "89.764px",
+    "letterSpacing": "-1.5px"
+  },
+  "h1-700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "112px",
+    "fontSize": "89.76px",
+    "letterSpacing": "-1.5px"
+  },
+  "h2-400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "72px",
+    "fontSize": "67.34px",
+    "letterSpacing": "-0.5px"
+  },
+  "h2-700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "72px",
+    "fontSize": "67.34px",
+    "letterSpacing": "-0.5px"
+  },
+  "h3-400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "56px",
+    "fontSize": "50.517px",
+    "letterSpacing": "0em"
+  },
+  "h3-700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "56px",
+    "fontSize": "50.52px",
+    "letterSpacing": "0em"
+  },
+  "h4-700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "40px",
+    "fontSize": "37.897px",
+    "letterSpacing": "0.25px"
+  },
+  "h4-400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "40px",
+    "fontSize": "37.897px",
+    "letterSpacing": "0.25px"
+  },
+  "h5-400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "32px",
+    "fontSize": "28.43px",
+    "letterSpacing": "-0.18px"
+  },
+  "h5-700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "32px",
+    "fontSize": "28.43px",
+    "letterSpacing": "-0.18px"
+  },
+  "h6-400": {
+    "fontFamily": "Inter",
+    "fontWeight": 400,
+    "lineHeight": "24px",
+    "fontSize": "21.328px",
+    "letterSpacing": "0px"
+  },
+  "h6-700": {
+    "fontFamily": "Inter",
+    "fontWeight": 700,
+    "lineHeight": "24px",
+    "fontSize": "21.33px",
+    "letterSpacing": "0px",
```

REMOVE  trailing comma
```suggestion
    "letterSpacing": "0px"
```